doc/rados/operations/monitoring.rst

```diff
@@ -655,3 +655,84 @@ Runtime`_.
 
 .. _Viewing a Configuration at Runtime: ../../configuration/ceph-conf#viewing-a-configuration-at-runtime
 .. _Storage Capacity: ../../configuration/mon-config-ref#storage-capacity
+
+Messenger Status
+=================
+
+Ceph daemons and librados clients support an admin socket command
+``messenger dump`` that allows access to a snapshot of runtime
+information about connections, sockets, bound addresses, and kernel
+TCP stats (via tcp(7) TCP_INFO).
+
+.. note:: The queried messenger needs to lock the connection data
+  structures for the time it takes to create the snapshot. This might
+  interfere with normal operation. Use the ``dumpcontents`` argument
+  to limit data structures dumped.
+
+Examples
+---------
+
+Without specifying a messenger return the available messengers:
```

Comment:
When a command is issued without specifying a messenger to dump, the list of available messengers is returned: